src/utils/walk-template.js

```diff
@@ -24,6 +24,12 @@ const walkTemplate = (
     const filePath = path.join(templateRootPath, fileName);
 
     if (isDirectory(filePath)) {
+      // Strip out the path where the user calls the generator so its not
+      // considered when ignoring folders
+      const strippedPath = filePath.slice(filePath.indexOf('create-using-template/node_modules/'));
```

Comment:
cant you instead slice based upon the templateRootPath? Its odd to see node_modules here, and although I understand (I think) why its here, if we could use the templateRootPath it would be more relocatable

Reply:
Good idea, this made it a lot cleaner.